packages/code-du-travail-data/Dockerfile

```diff
@@ -17,8 +17,6 @@ WORKDIR /app
 
 COPY ./package.json /app/package.json
 COPY --from=cdtn-base-image /app/packages/code-du-travail-data/dist /app/dist
-COPY ./dataset/stop_words/stop_words.json ./dataset/stop_words/stop_words.json
-COPY ./dataset/synonyms/synonyms.json ./dataset/synonyms/synonyms.json
```

Comment:
should you `COPY infra/elasticsearch/package.json` ?

Reply:
done above :)

Reply:
sorry i don't understand but l19 you only copy dist from data not the infra ?

Reply:
Ah sorry I looked at wrong file 😓

Reply:
This Dockerfile uses a bundled version of the sources. All dependencies are in the `dist` folder. We don't have to manually deal with them.

Reply:
> should you `COPY infra/elasticsearch/package.json` ?

The `COPY infra/elasticsearch/package.json` is in the "main" Dockerfile which builds the sources